ops/run-007-research/README.md

```diff
@@ -0,0 +1,68 @@
+# Run 007: Core-Geth Behavior Research and Network Sync Analysis
+
+## Issue Reference
+- **GitHub Issue**: run-009
+- **Date**: 2025-12-04
+
+## Executive Summary
+
+This investigation was triggered by concerns that removing the `looksLikeRLP` heuristic from MessageCodec negatively impacted peer connectivity. After comprehensive research of core-geth's source code, review of Fukuii's ADRs, and analysis of previous run documentation, the findings conclusively show:
+
+**The removal of `looksLikeRLP` was correct and necessary.** The current implementation is superior to previous versions. Peer connectivity issues at genesis are NOT related to compression but are instead due to the bootstrap challenge, which is already solved by bootstrap checkpoints (CON-002).
+
+## Key Findings
+
+### Finding 1: looksLikeRLP Removal Was CORRECT
```

Comment:
i agree removing this is correct, however core geth sending compressed hello ping and pong messages is interesting. we need to research why this would happen on the mainnet by reviewing the core geth logic and aligning our implementation to match